spec/google_coordinates_test.rb

```diff
@@ -0,0 +1,17 @@
+require_relative '../lib/google_coordinates.rb'
+
+describe "google coordinates functionality" do
+  # better tests would mock the API call
+  # todo: add tests for failed API call
+  it "returns the correct coordinates for 'Washington, DC'" do
+    expect(GoogleCoordinates.location_info('Washington, DC')).to eq(['Washington, DC, USA', '38.9071923,-77.0368707'])
```

Comment:
1. Because the tests here don't actually pass without mocking when the whole test suite is run together (ha! working on it...).
2. Because the tests should be testing the application itself, and reliable testing requires controlling the input to the application from external sources. If the test fails because Google suddenly returns a different longitude and latitude for Washington DC, that's not due to a failure of the darksky-slackbot application. (In this particular case, I wasn't worried about Google returning different coordinates.)
3. Because calling APIs can have consequences. In the Darksky case, they'd start charging  over 1000 requests. For other cases, you might want test "put" requests, for example, without actually sending them and changing anything.